smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/KafkaConnector.java

```diff
@@ -79,6 +82,8 @@
 
     public static final String CONNECTOR_NAME = "smallrye-kafka";
 
+    public static final Tracer TRACER = OpenTelemetry.getTracerProvider().get("io.smallrye.reactive.messaging.kafka");
```

Comment:
so even if tracing is disabled we instantiate the tracer? Does it make sense?

Reply:
There's a small cost yes, even to have the No Op tracer.

However, not setting it like this then means more work to ensure we don't create multiple versions later.

I don't mind doing that, but that would potentially slow things down a bit on the first use

Reply:
ok, make sense.